components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -2616,6 +2616,119 @@ paths:
                 $ref: '#/components/schemas/Error'
       tags:
         - Organization Application Sharing
+  /applications/loginflow/generate:
+    post:
+      tags:
+        - LoginFlow
+      summary: Initiate login flow generation
+      description: >
+        This API provides the capability to initiate the generation of a login flow.
+        <br>
+        <b>Permission required:</b>
+        * /permission/admin/manage/identity/applicationmgt/update
+        <br>
+        <b>Scope required:</b>
+        * internal_application_mgt_update
+      operationId: generateLoginFlow
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/LoginFlowGenerateRequest'
+      responses:
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LoginFlowGenerateResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /applications/loginflow/status/{operation_id}:
+    get:
+      tags:
+        - LoginFlow
+      summary: Get the status of the login flow generation process
+      operationId: getLoginFlowStatus
+      parameters:
+        - name: operation_id
```

Comment:
Be consistent in the naming convention. The rest of the places seems to be using camel case